docs/guides/graphics_tutorial.rst

```diff
@@ -0,0 +1,279 @@
+=============================================
+Exploring the Graphical capabilities of Rootex
+=============================================
+
+This documentation aims to showcase the graphical capabilities of rootex and act as a tutorial for beginners to get started.
+
+Let's start by creating a scene.
+
+Create a scene
+***************
+
+To Create a Scene 
+
+1) Go to `file->CreateScene`.
+
+2) Name the scene and click create.
+
+.. image:: images/graphics_tutorial/create_scene.png
+	:alt: Create Scene
+	:align: center
+
+|
+
+Now we Create an Empty Scene. An empty scene is nothing but objects. You can have different components in it, more on that later.
+
+Create Empty scene
+******************
+
+To create an empty scene.
+
+1) Right-click the root scene.
+2) Click Add Empty Scene
+
+.. image:: images/graphics_tutorial/add_empty_scene.png
+	:alt: Create Empty Scene
+	:align: center
+
+.. image:: images/graphics_tutorial/empty_scene_created.png
+	:alt: Empty Scene Created
+	:align: center
+
+|
+
+*****************
+Giving Components
+*****************
+
+Now we give components to the empty scene. 
+
+1) Right-click the empty scene.
+2) Click Edit Components.
+3) Check the appropriate components, in this case, transform and Model. **Note**: Transform Component is a must.
+
+.. image:: images/graphics_tutorial/add_model_component_and_transform_component.png
+	:alt: Components Added
+	:align: center
+	
+4) Open inspector
+5) Go to the model component in the inspector.
+6) Click the folder icon next to Model.
```

Comment:
Add an instruction telling the path of the sponza file to go to.

Reply:
It is visible in the screenshot

Reply:
It's better that it is written in the text too. Else, you'd have to zoom in to look at the filepath.